fastlane/Fastfile

```diff
@@ -0,0 +1,55 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+PODSPEC_FILE_PATHS = [
+  "Bow.podspec",
+  "BowEffects.podspec",
+  "BowOptics.podspec",
+  "BowRx.podspec",
+  "BowFree.podspec",
+  "BowGeneric.podspec",
+  "BowRecursionSchemes.podspec",
+  "BowGenerators.podspec",
+  "BowEffectsGenerators.podspec",
+  "BowRxGenerators.podspec",
+  "BowFreeGenerators.podspec",
+  "BowLaws.podspec",
+  "BowEffectsLaws.podspec",
+  "BowOpticsLaws.podspec",
+].freeze
```

Comment:
@truizlop I added the files in the order you mentioned

Reply:
Thanks!